core/templates/dev/head/domain/exploration/EditableExplorationBackendApiService.js

```diff
@@ -0,0 +1,163 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to send changes to a exploration to the backend.
+ */
+
+oppia.factory('EditableExplorationBackendApiService', [
+  '$http', '$q', 'EXPLORATION_DATA_URL_TEMPLATE',
+  'EDITABLE_EXPLORATION_DATA_URL_TEMPLATE', 'UrlInterpolationService',
+  'ReadOnlyExplorationBackendApiService',
+  function($http, $q, EXPLORATION_DATA_URL_TEMPLATE,
+    EDITABLE_EXPLORATION_DATA_URL_TEMPLATE, UrlInterpolationService,
+    ReadOnlyExplorationBackendApiService) {
+    var _fetchExploration = function(
+        explorationId, applyDraft, v, successCallback, errorCallback) {
```

Comment:
v is version in this case. In a few places its referred to as v and in some version. I'm thinking I will just change this to version.